docs/guides/dto-validation.md

```diff
@@ -96,6 +96,18 @@ create(@Body() createUserDto: CreateUserDto): any {
 }
 ```
 
+```typescript title="src/main.ts"
+import { I18nValidationExceptionFilter } from 'nestjs-i18n';
+
+  app.useGlobalFilters(new I18nValidationExceptionFilter());
```

Comment:
Think it would better to show the option right away (also in the info thingy)

```typescript
app.useGlobalFilters(new I18nValidationExceptionFilter({detailedErrors: true}));
```